{"pageProps":{"page":{"slug":"class.outofboundsexception.md","content":"# Клас OutOfBoundsException\n\n(PHP 5 >= 5.1.0, PHP 7, PHP 8)\n\n## Вступ\n\nВикидається виняток, якщо значення не є дійсним ключем. Це є помилками, які не можуть бути виявлені під час компіляції.\n\n## Огляд класів\n\n```classsynopsis\n\n    \n     class OutOfBoundsException\n    \n\n    \n     extends\n      RuntimeException\n     {\n\n    /* Наследуемые свойства */\n    \n      protected\n      string\n       $message = \"\";\nprivate\n      string\n       $string = \"\";\nprotected\n      int\n       $code;\nprotected\n      string\n       $file = \"\";\nprotected\n      int\n       $line;\nprivate\n      array\n       $trace = [];\nprivate\n      ?Throwable\n       $previous = null;\n\n\n    /* Наследуемые методы */\n    \n   public Exception::__construct(string $message = \"\", int $code = 0, ?Throwable $previous = null)\n\n    final public Exception::getMessage(): string\nfinal public Exception::getPrevious(): ?Throwable\nfinal public Exception::getCode(): int\nfinal public Exception::getFile(): string\nfinal public Exception::getLine(): int\nfinal public Exception::getTrace(): array\nfinal public Exception::getTraceAsString(): string\npublic Exception::__toString(): string\nprivate Exception::__clone(): void\n\n   }\n```\n","title":"Клас OutOfBoundsException","contentType":2,"navigation":[{"class.logicexception.md":"« LogicException"},{"class.outofrangeexception.md":"OutOfRangeException »"},{"index.md":"PHP Manual"},{"spl.exceptions.md":"Винятки"}]}},"__N_SSG":true}